projet/projects/urls.py

```diff
@@ -69,4 +65,39 @@
         views.expense_detail_update_delete,
         name="expense-detail-delete",
     ),
+    path("personal-budget/", views.create_personal_budget, name="personal-budget"),
+    path(
+        "update-budget/<int:budget_id>/",
+        views.update_get_delete_personal_budget,
+        name="something",
+    ),
+    path("all-personal/", views.list_users_personal_budgets),
+    path("create-expense/<int:budget_id>/", views.create_personal_expense),
+    path(
+        "create-feature/<int:project_id>/",
+        views.CreateFeature.as_view(),
+        name="feature-create",
+    ),
+    path(
+        "feature-delete/<int:feature_id>/", views.delete_feature, name="delete-feature"
+    ),
+    path(
+        "swagger-ui/",
+        TemplateView.as_view(
+            template_name="swagger-ui.html",
+            extra_context={"schema_url": "projetapi-schema"},
+        ),
+        name="swagger-ui",
+    ),
+    path("history/<int:project_id>/", views.history),
```

Comment:
Previously we couldn't get all the history of one project, now we've implemented that feature.